content/content/types/objects.md

```diff
@@ -34,7 +34,7 @@ assert carl.dead
 
 Object types are declared in a type section, as usual. They can be exported, and individual fields can also be exported. Fields can be safely exported without violating encapsulation because call syntax is equivalent between them.
 
-Initially, `carl` is created on the stack and initialized to zeros, so its value is `[name = nil, species = nil, age = 0]`. It is mutable, so that means that the contents of `carl` can be changed. This also means it can be passed to functions that require a variable parameter, like `sleep()`, which can modify its value.
+Initially, `carl` is created on the stack and initialized to zeros resp. empty string, so its value is `[name = "", species = "", age = 0]`. It is mutable, so that means that the contents of `carl` can be changed. This also means it can be passed to functions that require a variable parameter, like `sleep()`, which can modify its value.
```

Comment:
What does "resp." mean? Can you write it out?

Reply:
it is short for respectively, but as far as I know it is almost always abbreviated. But it should not be infix, so I rewrote the change